docs/installation/docker.md

```diff
@@ -1,10 +1,69 @@
 # Docker
-SigLens can be installed on Linux or macOS machine. 
-<br/>
 
-To install using Docker you need to run the following command:
+If you prefer to use Docker for SigLens installation on your Linux or macOS machine, follow this straightforward "one command" guide. Docker provides a convenient and isolated environment for running SigLens.
 
-```
+Installing SigLens with Docker streamlines deployment, encapsulating all dependencies within a container. This method is user-friendly, ensuring consistent environments and efficient resource usage. It's perfect for those who want a straightforward, hassle-free installation.
+
+## System Requirements
+
+Before proceeding, ensure you have Docker installed on your system:
+
+- **Docker:** Install Docker on your machine.
+  - [Install Docker](https://docs.docker.com/get-docker/)
+
```

Comment:
They don;t need this "pre-required" step of installing docker. Our script already takes care of installing docker if it is not present